packages/react-components/react-drawer/library/src/shared/drawerMotions.ts

```diff
@@ -14,22 +17,37 @@ const durations: Record<NonNullable<DrawerBaseProps['size']>, number> = {
   full: motionTokens.durationUltraSlow,
 };
 
+function getPositionTransform(
+  position: DrawerBaseProps['position'],
+  sizeVar: string,
+  dir: FluentProviderContextValue['dir'],
+) {
+  switch (position) {
```

Comment:
lets not use switch patterns as there are better and more human readable patterns to do this. lets go with simple if + early exit